### PR TITLE
docs: cache-redis breaking changes

### PR DESCRIPTION
**Pull request by olivermrbl**

### CODE REVIEW

File: www/apps/docs/content/upgrade-guides/modules/cache-redis/1-9-0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## How to Update
+
+Run the following command in your project:
+
+```bash npm2yarn
+npm install @medusajs/cache-redis@1.9.0
+```
+
+---
```
Comment (shahednasser) [inline www/apps/docs/content/upgrade-guides/modules/cache-redis/1-9-0.md]: This is not necessary since there's no section after it so you can remove it.

**Review by shahednasser — CHANGES_REQUESTED** [review]
One comment + we should add the new category to the sidebar in `sidebars.js`.

**Comment by olivermrbl** [comment]
Not sure where to add it in sidebars.js. I can't seem to find the product module one there either. Can you provide some guidance?

**Review by shahednasser — APPROVED** [review]
Sorry i actually mixed it up 😅 thought you were adding a new sidebar category.

LGTM

**Pull request merged**